we-smooth-fe/src/components/auth-context/login.tsx

```diff
@@ -20,7 +20,7 @@ const Login: FunctionComponent = () => {
     }, [])
 
     return (
-        <>
+        <div style={{ position: 'absolute', top: '65px' }}>
```

Comment:
at some point we should give this a className and have stylise it scss file

Reply:
Yes, of course. Given that the login page component is currently being designed, I've decided to hold off on adding the SCSS styling for now. Once its design is ready, we can proceed with implementing the component and its styles.